app-repo.json

```diff
@@ -1,58 +1,22 @@
 {
-  "name": "Pyoncord App Repo",
-  "identifier": "io.github.pyoncord.repo",
-  "iconURL": "https://pyoncord.github.io/assets/icon-altstore.png",
+  "name": "Revenge App Repo",
+  "identifier": "io.github.revenge-mod.repo",
+  "iconURL": "https://cdn.discordapp.com/attachments/1205207690352005243/1284228073415249950/icon-altstore.png?ex=66e5de18&is=66e48c98&hm=737f59f228f7d887881fff1fcb6d8b08258b91c3c0d4df8501dfc499b51d870d&",
```

Comment:
This needs to be changed. I'll have to create a branding repository. For now this can stay, I suppose.

Reply:
once you make the branding repo ill change and pr